bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/configurable/ConfigResolver.java

```diff
@@ -113,6 +113,8 @@ public ConfigResolver(Map<Module, VariableKey[]> configVarMap, RuntimeDiagnostic
             case TypeTags.RECORD_TYPE_TAG:
                 return getConfigValue(key, configProvider -> configProvider
                         .getAsRecordAndMark(module, key));
+            case TypeTags.XML_TEXT_TAG:
+                return getConfigValue(key, configProvider -> configProvider.getAsXmlAndMark(module, key));
```

Comment:
```suggestion
                return getConfigValue(key, configProvider -> configProvider
                        .getAsXmlAndMark(module, key));
```
Shall we use the same format used in the class?

Reply:
Also in L138